### ISSUE TITLE
 FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.

### ISSUE THREAD

**Issue by inphitude** [suno-ai/bark#615]
I keep receiving this issue when I try to generate audio. Does anyone know how to fix it?
C:\Users\Admin\AppData\Local\Programs\Python\Python312\Lib\site-packages\torch\nn\utils\weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
